daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2416,6 +2417,15 @@ func (devices *DeviceSet) UnmountDevice(hash, mountPath string) error {
 	}
 	logrus.Debug("devmapper: Unmount done")
 
+	// Remove the mountpoint here. This is necessary to avoid cases where a
```

Comment:
It might be good to tell that removing directory will remove the mount point on this directory from all the mount namespaces.